hyperspy/_components/gaussian2d.py

```diff
@@ -55,14 +55,13 @@ class Gaussian2D(Expression):
         Location of the Gaussian maximum (peak position) in `x` direction.
     centre_x : float
         Location of the Gaussian maximum (peak position) in `y` direction.
-    
-    
+
```

Comment:
The double line break is needed here. Otherwise, the following comment will be formatted as if it is another Parameter!